src/rpc/net.cpp

```diff
@@ -260,15 +260,18 @@ UniValue listbanned(const UniValue& params, bool fHelp)
 
     banmap_t banMap;
     g_banman->GetBanned(banMap);
+    const int64_t current_time{GetTime()};
```

Comment:
This is ok. I noticed Bitcoin had the argument that GetTime() is technically deprecated now, but this is not the place to change that out. We need to replace all of the GetTime() calls but that should be done in a separate PR.

Reply:
#2330 enables the replacement of legacy GetTime() calls.